src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
> I am concerned about the possibility of future giveaway patches before concerns about functionality are addressed.

That does not make sense to me. If a patch makes the code unmaintable, it can be rejected without investigating possible improvements. If it doesn't, you can judge it based on its functionality and code style, and if there is a concern raise it.

So in which case will the PR be accepted/rejected: If your experiments are successful or not, and why?

I can deal with criticism, but not with arbitrariness.